bin/oscapd-evaluate

```diff
@@ -103,6 +103,9 @@ def cli_scan(args, config):
     scanned_targets = []
     failed_targets = []
 
+    if not args.profile.startswith("xccdf_org.ssgproject.content_profile_"):
+        args.profile = "xccdf_org.ssgproject.content_profile_" + args.profile
```

Comment:
will this even do anything? it's changing parsed arguments. I don't like this code because it's hard for me to figure out what it will do.

what about users using custom non SSG content?